extensions/wikia/CommunityPage/scripts/ext.communityPage.js

```diff
@@ -165,20 +166,19 @@ require([
 		event.preventDefault();
 	});
 
-	$(document).on( 'click', '#modalTabAll', function (event) {
-		switchCommunityModalTab(tabs.TAB_ALL);
-		event.preventDefault();
-	});
-
-	$(document).on( 'click', '#modalTabAdmins', function () {
-		switchCommunityModalTab(tabs.TAB_ADMINS);
-		event.preventDefault();
-	});
-
-	$(document).on( 'click', '#modalTabLeaderboard', function () {
-		switchCommunityModalTab(tabs.TAB_LEADERBOARD);
-		event.preventDefault();
-	});
+	$(document)
+		.on( 'click', '#modalTabAll', function (event) {
+			switchCommunityModalTab(tabs.TAB_ALL);
```

Comment:
@kamilkoterba and @perjg moved the `event.preventDefault();` line above. You could move it as well.